src/fixture.c

```diff
@@ -168,7 +169,7 @@ struct io
     unsigned randomized_election_timeout; /* Value returned by io->random() */
     unsigned network_latency;             /* Milliseconds to deliver RPCs */
     unsigned disk_latency;                /* Milliseconds to perform disk I/O */
-    unsigned work_duration;               /* Milliseconds to long running work */
+    unsigned work_duration; /* Milliseconds to long running work */
```

Comment:
It was just because the line was too long. I reworded the comment, which also reads better now:

```C
    unsigned randomized_election_timeout; /* Value returned by io->random() */
    unsigned network_latency;             /* Milliseconds to deliver RPCs */
    unsigned disk_latency;                /* Milliseconds to perform disk I/O */
    unsigned work_duration;               /* Milliseconds to run async work */
```